lib/beaker/hypervisor/vmpooler.rb

```diff
@@ -19,6 +19,18 @@ def initialize(vmpooler_hosts, options)
       @options = options
       @logger = options[:logger]
       @hosts = vmpooler_hosts
+      @credentials = load_credentials(@options[:dot_fog])
+    end
+
+    def load_credentials(dot_fog = '.fog')
```

Comment:
It's actually loading from the `@options[:dot_fog]` location, this is just a default (stolen almost completely from the AWS hypervisor).